src/main/java/core/basesyntax/service/ReaderService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface ReaderService {
```

Comment:
Follow one type of code style in naming, if you have `FileWriterService` then create `FileReaderService`